stout_a05a.js

```diff
@@ -0,0 +1,29 @@
+/*Kyle Stout
+  stout_a05a
+  INFO 2124WW
+  Thoendel
+  1-19-2020
+*/
+
+let todolist = ["Eat", "Sleep", "Shower", "Brush Teeth"];
+todolist;
+todolist.unshift("Wake up");
+todolist;
+todolist.reverse();
+todolist.sort();
+todolist;
+todolist.includes("x");
+
+let grocerylist = new Array();
+grocerylist[0] = 'Milk';
+grocerylist[1] = 'Bread';
+grocerylist[2] = 'Sugar';
+grocerylist[3] = 'Flour';
+grocerylist[4] = 'Salt';
+grocerylist[5] = 'Cookies';
+grocerylist;
+grocerylist.shift();
+grocerylist.splice(4,0, 'Crackers');
+grocerylist.splice(4,0, 'Tea');

```

Comment:
add 2 elements in the middle. at this point you have 5 values - so adding at index 2 or 3 would make more sense.